autopush/tests/test_integration.py

```diff
@@ -183,6 +188,9 @@ def get_notification(self, timeout=0.2):
         except:
             return None
 
+    def get_notification(self, timeout=0.2):
```

Comment:
I wanted `send_notification` to accept a timeout, but I couldn't call `self.get_notification` directly—that caused  `__getattribute__` to wrap it. I also made a change on line 74 to ignore all `_`-prefixed methods. Is there a better way to bypass `__getattribute__`?